src/GlobalStyles.js

```diff
@@ -2,18 +2,87 @@ import { createGlobalStyle } from 'styled-components'
 
 export default createGlobalStyle`
     
- * {
+* {
         box-sizing: border-box;
-    }
+}
```

Comment:
Falls dich die margin, die der Browser in seinen default stlyes hat auch stören sollte, kann ich dir empfehlen hier ein `margin: 0;` zu definieren. :)